# HG changeset patch
# User doug@doughellmann.com
# Date 1443045772 0
# Node ID c998cc18cd47fbd877d54b127dca419a5fa455c0
# Parent  8705608cb3aee16c135057cb9ca4413a61cc069d
Use sys.platform instead of subprocess.mswindows

Because subprocess.mswindows was renamed to reflect the fact that it is
meant to be a private symbol, use the public sys.platform to detect when
we are running on Windows.

Fix #18

diff --git a/cmd2.py b/cmd2.py
--- a/cmd2.py
+++ b/cmd2.py
@@ -184,7 +184,7 @@
 to be installed on operating system.
 %s"""
 
-if subprocess.mswindows:
+if sys.platform == 'win32':
     try:
         import win32clipboard
         def get_paste_buffer():
